assets/app/view/game/dividend.rb

```diff
@@ -70,31 +70,63 @@ def render
           ])
         end
 
+        div_props = {
+          key: 'dividend',
+          hook: {
+            destroy: -> { cleanup },
+          },
+        }
+
         table_props = {
           style: {
             margin: '0.5rem 0 0 0',
             textAlign: 'left',
           },
-          key: 'dividend',
-          hook: {
-            destroy: -> { cleanup },
-          },
         }
         share_props = { style: { width: '2.7rem' } }
 
-        h(:table, table_props, [
-          h(:thead, [
-            h(:tr, [
-              h('th.no_padding', 'Dividend'),
-              h(:th, 'Treasury'),
-              h(:th, share_props, 'Per Share'),
-              h(:th, 'Stock Moves'),
+        if corporation_interest_penalty?(entity)
```

Comment:
does this file need to change?

Reply:
Changing it lets us warn the acting president that there are consequences to their interest and it also clearly shows why they are paying out less than they calculated a moment before in the train routing phase
![image](https://user-images.githubusercontent.com/2993555/101387267-34f54c00-388c-11eb-8fbd-9402d7fc4d58.png)